Sources/CoreCommands/SwiftTool.swift

```diff
@@ -395,6 +395,10 @@ public final class SwiftTool {
             return workspace
         }
 
+        if options.resolver.skipDependencyUpdate {
+            self.observabilityScope.emit(warning: "'--skip-update' option is deprecated and will be removed in a future release")
```

Comment:
Is there a replacement for `--skip-update`? What exactly should be used instead of it?

Reply:
There's no replacement for it since it never really did what people wanted. Eventually maybe something like https://github.com/apple/swift-package-manager/issues/4705 could be used as a replacement.